### DISCUSSION TITLE
Database selection for the Zenith application

### DISCUSSION THREAD

**Discussion by Meminseeker** [bounswe/bounswe2024group11#156] [Polls]
Please express your ideas about your choice below. We may discuss some aspects.

**Comment by Meminseeker** [comment]
@MucahitErdoganUnlu @umitcan07 @ceydanursen @Meminseeker @meminciftci @tenacke @hks1444 @ArdaVural @yunuskaydin @ozankrkya

**Comment by MucahitErdoganUnlu** [comment]
We already have familiarity with MySQL. We have setup on our computers as well. I think it will be wiser to use MySQL

**Reply by Meminseeker** [reply]
I agree with you about this. It would be easier for us to adapt to MySQL since both we use it in Cmpe 321 class and it is known for its easy-to-use experience. 

On the other hand, PostgreSQL is mighty and reliable, and I see many global and innovative projects rely on PostgreSQL. Also, according to the StackOverflow survey, PostgreSQL is 10% more used than MySQL among professional developers. ([go check it out](https://survey.stackoverflow.co/2023/#section-most-popular-technologies-databases)) It also has a very assertive admired-desired table result. ([go check it out](https://survey.stackoverflow.co/2023/#section-admired-and-desired-databases))


I'm not a fan of either; I want to express the trade-off here. We may continue with MySQL, as it is more than enough for this project, or we may continue with PostgreSQL, as it is more promising for the future with some additional efforts. What do you think about this?

**Reply by ArdaVural** [reply]
I think it is better to use MySQL because learning cost is less and when we think about our plans, I think we can spend that learning time on something else.
Also, as you said, Even if PostgreSQL is more promising for the future, MySQL is more than enough for our own project.
Still, we should consider others' opinions aswell.

**Comment by Meminseeker** [comment]
I'm closing this discussion since we decided to our database as MySQL
